src/core/indexedlist.h

```diff
@@ -44,34 +57,42 @@ class IndexedList
     {
         return v.begin();
     }
+
     inline const_iterator begin() const
     {
         return v.begin();
     }
+
     inline const_iterator cbegin() const
     {
         return v.cbegin();
     }
+
     inline iterator end()
     {
         return v.end();
     }
+
     inline const_iterator end() const
     {
-        return v.end();
+        return v.cend();
```

Comment:
Not exactly, but the return value is wrong (it's a typedef). Will change to non-const iterator.